internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/multicluster/TestMetadataOperationInMultiCluster.java

```diff
@@ -36,7 +36,7 @@
 public class TestMetadataOperationInMultiCluster {
   private static final Logger LOGGER = LogManager.getLogger(TestMetadataOperationInMultiCluster.class);
 
-  @Test(timeOut = 60 * Time.MS_PER_SECOND)
+  @Test(timeOut = 180 * Time.MS_PER_SECOND)
```

Comment:
This test kept timing out in the CI and this fixed it, unsure if it's related to the issues with `DaVinciClusterAgnosticTest`